contributing.md

```diff
@@ -1,9 +1,109 @@
+everybody to be able to contribute to Directus, no matter your background or expertise. In order to facilitate that,
+we've put together a couple tips and tricks below. Our team truly appreciates every single contributor, community
+member, GitHub star, pull-request, bug report, and feature request. Keeping Directus completely free and open-source is
+our way of saying: **Thank you!**
 
-Please note we have a [code of conduct](./code_of_conduct.md), please follow it in all your interactions with the
+> We're here to help!
+>
+> If you have _any_ questions along your contributor journey, please feel free to come chat with us on
+> [our Discord server](https://directus.chat).
+
+## Code of Conduct
+
+**The Directus [Code of Conduct](https://github.com/directus/directus/blob/main/code_of_conduct.md) is one of the ways
+we put our values into practice. We expect all of our staff, contractors and contributors to know and follow this
+code.**
+
+**Our contributors and maintainers work extremely hard to build Directus as premium open-source software. Please be
+respectful of those efforts throughout our ecosystem. Trolling, harassing, insulting, or other unacceptable behavior by
+participants will not be tolerated.**
+
+In the interest of fostering an open and welcoming environment, we as pledge to make participation in our project and
+community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex
+characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality,
+personal appearance, race, religion, or sexual identity and orientation.
+
+Examples of behavior that contributes to creating a positive environment include:
+
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy towards other community members
+
+**Before continuing, please take a moment to read our full
+[Code of Conduct](https://github.com/directus/directus/blob/main/code_of_conduct.md).**
+
+## Ways to Contribute
+
+### Reporting Bugs
+
+If you happen to run into a bug, please post an Issue on our main GitHub Issue board:
+https://github.com/directus/directus/issues
+
+Please be as explicit and detailed as you can in the bug report. The more information is available, the easier it is for
+other contributors to help you find the solution or fix.
+
+### Leaving Feedback
+
+If you have a great idea for an improvement of the platform, or any other feedback, please make sure to open a new
+Discussion on our GitHub Discussions board: https://github.com/directus/directus/discussions
+
+### Document the Project
+
+[The Directus Docs](https://github.com/directus/docs) are a living document that can always be improved on. Any parts of
+the docs in dire need of some tender love and care? Feel free to open a Pull Request!
```

Comment:
```suggestion
[The Directus Docs](https://github.com/directus/docs) are a living document that can always be improved on. Notice any parts of
the docs in dire need of some tender love and care? Feel free to open a Pull Request!
```

Not sure if this "Notice" (or any other alternatives) is necessary, as it felt like something is missing before "Any". But could also just a phrase/structure that I'm not personally familiar with 😄